### PR TITLE
GOD

### PR DESCRIPTION
**Pull request by GATEK33pER**

### CODE REVIEW

**Comment by Santesprogtx** [comment]
+

**Comment by cpallin0** [comment]
add

**Comment by n1v2s3r4** [comment]
[" బ్లాక్ చైన్         fake  fake "]

File: src/api.js [hunk 1]
```diff
@@ -257,6 +257,63 @@ MerchantAPI.prototype.unarchiveAccount = function (guid, options) {
   })
 }
 
+// 📔 Contacts
+MerchantAPI.prototype.getWalletContacts = function (guid, options) {
+  return this.getWalletHD(guid, options).then(function (wallet) {
```
Comment (toya999) [inline src/api.js]: 1KU36fxVUskAUHqrxj5kqmuPW6W7k4v1P8

Comment (toya999) [reply src/api.js]: 1KU36fxVUskAUHqrxj5kqmuPW6W7k4v1P8

Comment (faxata) [inline src/api.js]: xpub6CQRJD6vK2p6HqxvBkQ477DX2zTcFMAtA3fV78SciybNtPJonh6VVGvXQgoWsP5opSp2vE2F5gn4H8dqhNrjju3zNyjNg7rq1cccuH1zz5x

Comment (faxata) [inline src/api.js]: 1JyuARYbS8rvuDxi73qpkaRc9aCcrYh7PC

**Pull request closed without merging**